src/javascripts/helpers/domEvents.js

```diff
@@ -0,0 +1,28 @@
+import printShows from '../components/printShows';
+import { buildSouvenirs } from '../components/buildSouvenirs';
+import getSouvenirs from './data/souvenirData';
+import showFood from '../components/showFood';
+import getFood from './data/foodData';
+import getStaff from './data/staffData';
+import { showStaff } from '../components/showStaff';
+import getShows from './data/showsData';
+
+const domEvents = () => {
+  document.querySelector('body').addEventListener('click', (e) => {
+    console.warn(e.target.id);
+    if (e.target.id.includes('souvenir-view')) {
+      getSouvenirs('-MUuao2lynNljpuXaHnJ').then((souvenirArray) => buildSouvenirs(souvenirArray));
```

Comment:
Not sure that I like this, but I know you are working on one event, but this really should be dynamic and not hardcoded, but go ahead and make a ticket to cover making these IDs dynamic for later

Reply:
Just added an issue ticket for this!  #61